bin/spice/cmd/trace.go

```diff
@@ -33,7 +33,7 @@ var (
 	trace_id string
 )
 
-var supported_trace_tasks = []string{"ai_chat", "ai_completion", "sql_query", "nsql",
+var supported_trace_tasks = []string{"ai_chat", "accelerated_refresh", "ai_completion", "sql_query", "nsql",
```

Comment:
Instead of hard-coding this list, why not just pull it from the task history table?

Reply:
IIRC we want to give better user messages when delineating between:
1. Trace is not in `runtime.task_history` because this event has not occured.
2. Trace will never be in `runtime.task_history` because its invalid/typo.